packages/client/hmi-client/src/components/workflow/tera-model-node.vue

```diff
@@ -52,15 +52,16 @@ const parameterValues = ref<StringValueMap>({});
 
 function run() {
 	if (ModelOperation.action) {
-		console.log(
-			ModelOperation.action({
-				model: model.value,
-				initialValues: initialValues.value,
-				parameterValues: parameterValues.value
-			})
-		);
+		// console.log(
+		// 	ModelOperation.action({
+		// 		model: model.value,
+		// 		initialValues: initialValues.value,
+		// 		parameterValues: parameterValues.value
+		// 	})
+		// );
 		emit('append-output-port', {
 			type: ModelOperation.outputs[0].type,
+			label: model.value?.name,
```

Comment:
Would it be wiser to omit this property if it's value is null?

Reply:
No not currently, I am checking for value === null explicitly in the service.